config/webpack.config.dev.js

```diff
@@ -73,7 +73,7 @@ module.exports = {
         loader: 'json'
       },
       {
-        test: /\.(jpg|png|gif|eot|svg|ttf|woff|woff2)$/,
+        test: /\.(jpg|png|gif|eot|svg|ttf|woff|woff2)(\?.*)?$/,
```

Comment:
I think it's relevant for anything that can be referenced in CSS. So CSS itself won't need it but any files will.